command/runner.go

```diff
@@ -61,6 +61,18 @@ func Run(args []string, version string) error {
 
 	shouldColorize, subcommandInfo := ResolveSubcommand(args, config)
 
+	subcmd := subcommandInfo.Subcommand
+	if config.Pager && isOutputTerminal() &&
+		(subcmd == kubectl.Get || subcmd == kubectl.Describe) {
```

Comment:
Why only "get" and "describe"?

Reply:
Commands like edit and attach breaks if you pipe to a pager. I thought it safer to have a whitelist. Looking through the commands, I think we can also add "logs" and "explain".

Reply:
Yeah sure, sound logic. We've had multiple issues in the past for going with a "blacklist" approach 😅